.teamcity/src/main/kotlin/model/FunctionalTestBucketProvider.kt

```diff
@@ -9,7 +9,6 @@ import jetbrains.buildServer.configs.kotlin.v2019_2.BuildStep
 import jetbrains.buildServer.configs.kotlin.v2019_2.BuildSteps
 import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.script
 import java.io.File
-import java.util.LinkedList
 
 const val MASTER_CHECK_CONFIGURATION = "Gradle_Master_Check"
```

Comment:
I think this constant should move to the Generator file. The next constant probably as well.

Reply:
Done.